examples/.bazelrc

```diff
@@ -0,0 +1,15 @@
+# Convenient flag shortcuts.
```

Comment:
Let's symlink this file, from top level, instead of copying it. Just like that in docs directory.

Reply:
Is the top-level `.bazelrc` even needed?